src/ape/cli/commands.py

```diff
@@ -14,5 +14,9 @@ class NetworkBoundCommand(click.Command):
 
     def invoke(self, ctx: Context) -> Any:
         value = ctx.params.get("network") or networks.default_ecosystem.name
-        with networks.parse_network_choice(value):
+        interactive = ctx.params.get("interactive")
+        # If not found, check the parent context.
+        if interactive is None and ctx.parent:
+            interactive = ctx.parent.params.get("interactive")
```

Comment:
what about `parent.parent` ? Should this be recursive?

Reply:
yeah the real goal is to see if any parent context has this flag set, I made this a recursive function now